src/textual/widget.py

```diff
@@ -368,20 +368,17 @@ def scrollbars_enabled(self) -> tuple[bool, bool]:
     @property
     def scrollbar_dimensions(self) -> tuple[int, int]:
         """Get the size of any scrollbars on the widget"""
-        return (
-            self._get_scrollbar_thickness_horizontal(),
-            self._get_scrollbar_thickness_vertical(),
-        )
+        return (self.scrollbar_size_horizontal, self.scrollbar_size_vertical)
 
     @property
-    def scrollbar_width(self) -> int:
+    def scrollbar_size_vertical(self) -> int:
```

Comment:
In retrospect the original was a poor name. It is the "width" of the "vertical" scrollbar, which is accurate, but muddles a few concepts.